### PR TITLE
Add .net6 to targets

### PR DESCRIPTION
**Pull request by pratiksanglikar**
Add `.NET 6` to targets

### CODE REVIEW

File: build/CallPowerShell.cmd [hunk 1]
```diff
@@ -0,0 +1 @@
+PowerShell %~dp0..\%*
```
Comment (jimmyca15) [inline build/CallPowerShell.cmd:1]: I see pack.cmd is removed, doesn't look like build.cmd is removed though. Any reason?

Comment (pratiksanglikar) [reply build/CallPowerShell.cmd:1]: Good catch, it was an oversight on my part.

**Review by jimmyca15 — APPROVED** [review]

File: src/Microsoft.Azure.AppConfiguration.Functions.Worker/Microsoft.Azure.AppConfiguration.Functions.Worker.csproj [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <Import Project="..\..\build\NugetProperties.props" />
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```
[the remark this answers is not in the document]
Comment (pratiksanglikar) [reply src/Microsoft.Azure.AppConfiguration.Functions.Worker/Microsoft.Azure.AppConfiguration.Functions.Worker.csproj:5]: Microsoft has officially ended the support for .NET 5 in May 2022, I think we are okay with having just .NET 6.
[.NET Core Support Cycles](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core)

[the remark this answers is not in the document]
Comment (jimmyca15) [reply src/Microsoft.Azure.AppConfiguration.Functions.Worker/Microsoft.Azure.AppConfiguration.Functions.Worker.csproj:5]: Good point, that needs a major version bump if we remove a target.

[the remark this answers is not in the document]
Comment (avanigupta) [reply src/Microsoft.Azure.AppConfiguration.Functions.Worker/Microsoft.Azure.AppConfiguration.Functions.Worker.csproj:5]: We can keep .net 5 support for now and plan for removing it from all packages in the next major release.

File: tests/Tests.AzureAppConfiguration.Functions.Worker/Tests.AzureAppConfiguration.Functions.Worker.csproj [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```
Comment (avanigupta) [inline tests/Tests.AzureAppConfiguration.Functions.Worker/Tests.AzureAppConfiguration.Functions.Worker.csproj:4]: Can you also add .net5 back here?

Comment (pratiksanglikar) [reply tests/Tests.AzureAppConfiguration.Functions.Worker/Tests.AzureAppConfiguration.Functions.Worker.csproj]: Done!

**Review by avanigupta — APPROVED** [review]

**Pull request merged**